core/src/main/scala/cats/syntax/representable.scala

```diff
@@ -0,0 +1,18 @@
+package cats
+package syntax
+
+trait RepresentableSyntax {
+  implicit final def catsSyntaxTabulate[A, R](f: R => A): TabulateOps[A, R] =
+    new TabulateOps[A, R](f)
+
+  implicit final def catsSyntaxIndex[F[_], A, R](fa: F[A])(implicit R: Representable.Aux[F, R]): IndexOps[F, A, R] =
+    new IndexOps[F, A, R](fa)
+}
+
+final class IndexOps[F[_], A, R](fa: F[A]) {
+  def index(implicit R: Representable.Aux[F, R]): R => A = R.index(fa)
+}
+
+final class TabulateOps[A, R](f: R => A) {
```

Comment:
Why not extend AnyVal here and above?

Reply:
If there is a reason can we comment?

Reply:
When I first wrote it, I had the ops class taking implicit params and AnyVal doesn't allow multiple param lists. I missed adding it back when I changed that, will fix that up now.